packages/perseus-editor/src/widgets/expression-editor.tsx

```diff
@@ -491,58 +493,62 @@ class AnswerOption extends React.Component<
 > {
     state = {deleteFocused: false};
 
-    handleDeleteBlur = () => {
-        this.setState({deleteFocused: false});
-    };
-
     change = (...args) => {
         return Changeable.change.apply(this, args);
     };
 
     render(): React.ReactNode {
-        let removeButton: React.ReactNode | null = null;
-        if (this.state.deleteFocused) {
-            removeButton = (
-                <button
-                    type="button"
-                    className="simple-button orange"
+        const removeButton = this.state.deleteFocused ? (
+            <>
+                <Button
+                    size="small"
                     onClick={this.handleImSure}
-                    onBlur={this.handleDeleteBlur}
+                    color="destructive"
                 >
                     I'm sure!
-                </button>
-            );
-        } else {
-            removeButton = (
-                <button
-                    type="button"
-                    className="simple-button orange"
-                    onClick={this.handleDelete}
-                >
-                    Delete
-                </button>
-            );
-        }
+                </Button>
+                <Strut size={spacing.small_12} />
+                <Button size="small" onClick={this.handleCancelDelete} light>
+                    Cancel
+                </Button>
+            </>
+        ) : (
+            <Button
+                size="small"
+                onClick={this.handleDelete}
+                color="destructive"
+                light
+            >
+                Delete
+            </Button>
+        );
+
+        const answerStatusCss = css(
+            styles.answerStatus,
+            this.props.considered === "wrong" && styles.answerStatusWrong,
+            this.props.considered === "correct" && styles.answerStatusCorrect,
+            this.props.considered === "ungraded" && styles.answerStatusUngraded,
+        );
 
         return (
-            <div className="expression-answer-option">
-                <div className="answer-handle" />
+            <div className={css(styles.answerOption)}>
+                <div className={css(styles.answerHandle)} />
 
-                <div className="answer-body">
-                    <div className="answer-considered">
-                        <div
+                <div className={css(styles.answerBody)}>
+                    <div>
+                        <button
```

Comment:
Does it make sense to migrate this to be a WB button right away (or at least a Clickable)?

Reply:
I tried to move it to Clickable (WB button I think would be too difficult to customize) and I ran into the most bizarre bug I've ever seen and now I'm tired and I don't want to fight it anymore so it's staying a `button`.

https://khanacademy.slack.com/archives/CEA6W0F6F/p1722982631694429